apps/payments/next/app/[locale]/[offeringId]/checkout/[interval]/[cartId]/start/page.tsx

```diff
@@ -158,6 +159,7 @@ export default async function Checkout({ params }: { params: CheckoutParams }) {
             </p>
           </div>
         </section>
+        <StripeWrapper amount={1099} currency="usd" />
```

Comment:
Should currency be hardcoded? Just temporarily?

Reply:
Yup, both are temporary. These will change to real values once we have the actual invoice data as part of the cart data.